.github/workflows/ci.yaml

```diff
@@ -94,6 +94,30 @@ jobs:
           version: v1.55.2
           working-directory: ${{ matrix.directory }}
 
+  govulncheck:
+    name: govulncheck
+    runs-on: ubuntu-22.04
```

Comment:
Most of our ci runs on `ubuntu-20.04` would be easier to stick to a single version